index.js

```diff
@@ -10,8 +10,14 @@ module.exports = {
         // Index page
         "page": function(page) {
             if (this.options.generator == 'website') {
+                // multiple languages, if configured
+                var lang = "";
+                if(this.context.config.language) {
```

Comment:
You should not use `this.context.config`, `this` is a `Book` instance,

it's better to do:

```js
var lang = this.isSubBook()? this.config.get('language') : null;
if (lang) lang = lang + '/';
```

We need to use `this.isSubBook()`, because the config `language` can be defined for a non-multilingual book.